### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by toinehartman**

### CODE REVIEW

File: src/main/java/programminglife/gui/controller/GuiController.java [hunk 1]
```diff
@@ -63,37 +60,64 @@
...
+    @FXML
+    private ResizableCanvas canvas;
+    @FXML
+    private AnchorPane anchorLeftControlPanel;
+    @FXML
+    private AnchorPane anchorGraphPanel;
+    @FXML
+    private AnchorPane anchorGraphInfo;
+    @FXML
+    private Canvas miniMap;
```
Comment (mvanmeerten) [inline src/main/java/programminglife/gui/controller/GuiController.java]: This is not very nice. @toinehartman

File: src/main/java/programminglife/gui/controller/GuiController.java [hunk 2]
```diff
@@ -472,44 +493,49 @@ private void initMouse() {
...
+     * @param y            coordinate where is clicked.
      * @param shiftPressed boolean if shift is pressed it should be displayed in panel 2.
      */
     private void mouseClick(double x, double y, boolean shiftPressed) {
         DrawableNode clickedOn = graphController.onClick(x, y);
         if (clickedOn != null) {
             if (clickedOn instanceof DrawableSegment) {
                 DrawableSegment segment = (DrawableSegment) clickedOn;
-                showInfoNode(segment, shiftPressed ? 240 : 10);
+                if (shiftPressed) {
```
Comment (mvanmeerten) [inline src/main/java/programminglife/gui/controller/GuiController.java]: Again, the previous code is more elegant

File: src/main/java/programminglife/gui/controller/GuiController.java [hunk 3]
```diff
@@ -632,17 +659,24 @@ void setText(int center, int radius) {
...
-        Text childrenText = new Text("Child: "); childrenText.setLayoutX(x); childrenText.setLayoutY(165);
+        Text idText = new Text("Genomes: ");
+        idText.setLayoutX(x);
+        idText.setLayoutY(65);
+        Text parentsText = new Text("Parent: ");
+        parentsText.setLayoutX(x);
+        parentsText.setLayoutY(115);
+        Text childrenText = new Text("Child: ");
+        childrenText.setLayoutX(x);
+        childrenText.setLayoutY(165);
```
Comment (mvanmeerten) [inline src/main/java/programminglife/gui/controller/GuiController.java]: Good change

File: src/main/java/programminglife/model/drawing/DrawableSNP.java [hunk 4]
```diff
@@ -9,8 +9,12 @@
 import java.util.stream.Collectors;
 
 /**
+<<<<<<< HEAD
+ * A subclass of {@link DrawableNode} representing Singular Nucleotide Polymorphism.
+=======
  * Class representing a SNP.
  * Contains from 2 to 4 nodes.
+>>>>>>> origin/develop
```
Comment (MeAmAnUsername) [inline src/main/java/programminglife/model/drawing/DrawableSNP.java]: This is a merge conflict

File: src/main/java/programminglife/model/drawing/DrawableNode.java [hunk 5]
```diff
@@ -69,65 +65,82 @@ public final int getIdentifier() {
...
 
     /**
+<<<<<<< HEAD
+=======
      * Set the size of this drawing.
      */
     protected abstract void setDrawDimensions(double zoomLevel);
 
     /**
+>>>>>>> origin/develop
```
Comment (MeAmAnUsername) [inline src/main/java/programminglife/model/drawing/DrawableNode.java]: And another

File: src/main/java/programminglife/gui/controller/GuiCreateBookmarkController.java [hunk 6]
```diff
@@ -17,12 +16,18 @@
...
+    @FXML
+    private Button btnCancel;
+    @FXML
+    private TextField txtBookmarkName;
+    @FXML
+    private TextField txtId;
+    @FXML
+    private TextField txtRadius;
+    @FXML
+    private TextArea txtDescription;
```
Comment (mvanmeerten) [inline src/main/java/programminglife/gui/controller/GuiCreateBookmarkController.java]: Again not very nice

File: src/main/java/programminglife/model/drawing/DrawableNode.java [hunk 7]
```diff
@@ -69,65 +65,82 @@ public final int getIdentifier() {
...
 
     /**
      * Color this according to contents.
+<<<<<<< HEAD
+     *
+     * @param subGraph  {@link SubGraph} to colorize.
+     * @param zoomLevel is double with a value which describes what the current zoom is.
+=======
      * @param subGraph {@link SubGraph} to colorize.
+>>>>>>> origin/develop
```
Comment (MeAmAnUsername) [inline src/main/java/programminglife/model/drawing/DrawableNode.java]: And here

File: src/main/java/programminglife/gui/controller/GuiLoadBookmarkController.java [hunk 8]
```diff
@@ -29,11 +28,16 @@
...
+    @FXML
+    private Button btnOpenBookmark;
+    @FXML
+    private Button btnDeleteBookmark;
+    @FXML
+    private Button btnCreateBookmark;
+    @FXML
+    private Button btnShowInfo;
+    @FXML
+    private Accordion accordionBookmark;
```
Comment (mvanmeerten) [inline src/main/java/programminglife/gui/controller/GuiLoadBookmarkController.java]: Same goes for this

File: src/main/java/programminglife/model/drawing/DrawableSNP.java (hunk 4 above)
Comment (mvanmeerten) [inline src/main/java/programminglife/model/drawing/DrawableSNP.java]: Merge conflict

File: src/main/java/programminglife/model/drawing/SubGraph.java [hunk 9]
```diff
@@ -863,36 +884,21 @@ private void mergeLeftSubGraphIntoThisSubGraph(SubGraph leftSubGraph) {
...
-        // when getting bigger: include new nodes
-        // when getting smaller: drop nodes outside new radius.
-    }
-
     public LinkedHashMap<Integer, DrawableNode> getNodes() {
         return this.nodes;
     }
 
-    public GenomeGraph getGraph() {
+    private GenomeGraph getGraph() {
```
Comment (mvanmeerten) [inline src/main/java/programminglife/model/drawing/SubGraph.java]: this should be public

**Pull request merged**